purchase_stock_picking_return_invoicing_open_qty/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================================
+Purchase Stock Picking Return Invoicing Open Qty
+================================================
+
+This module extends makes compatible the modules 'Purchase Open Qty' and
+'Purchase Stock Picking Return Invoicing'.
```

Comment:
Please indicate where can the module 'purchase_open_qty' be obtained from. (OCA/purchase-workflow)